Easydict/NewApp/View/MenuItemView.swift

```diff
@@ -8,16 +8,33 @@
 
 import Sparkle
 import SwiftUI
+import ZipArchive
 
 @available(macOS 13, *)
 struct MenuItemView: View {
+    private let updater: SPUUpdater
```

Comment:
Do we need to add `ObservableObject` here?

```swift
final class MenuItemStore: ObservableObject {
	let update: SPUUpdater
	init(updater: SPUUpdater) {
	}
}
```

```
struct MenuItemView: View {
	@StateObject var store = MenuItemStore()
}
```

Reply:
This is a very good method, and the official code examples are similar to this. However, this only checks for updates and does not involve listening to more status changes and UI changes. Of course, we need to listen to @tisfeng 
 suggestions on this. Personally, I think we can maintain the status.

Reply:
I'm not very familiar with Swift/SwiftUI, and that part of my knowledge is still a long way off. However, it does look like there is no need to listen for state changes here.

Reply:
I'd like to rewrite the code in such a way that the functionality should just be as consistent as possible with what it was before.

Also, I made some changes to [Check for Updates] before, and the `feedURLString` in the debug environment is a convenient URL for local debugging, which I'd like to add as well. @liyafly 

https://github.com/tisfeng/Easydict/blob/261b625f3d770365ff1fa0ddabe6a030b0592d08/Easydict/App/AppDelegate.m#L104

Reply:
Understood, I’ve grasped your requirements. I also aim to refactor the code into a more scalable architecture, while carefully comparing it with the previous functionalities to ensure a proper restructuring. I will complete the initial development within this week.